src/main/java/ch/njol/skript/conditions/CondMatches.java

```diff
@@ -19,11 +19,13 @@
 
 @Name("Matches")
 @Description("Checks whether the defined strings match the input regexes (Regular expressions).")
-@Examples({"on chat:",
-	"\tif message partially matches \"\\d\":",
-	"\t\tsend \"Message contains a digit!\"",
-	"\tif message doesn't match \"[A-Za-z]+\":",
-	"\t\tsend \"Message doesn't only contain letters!\""})
+@Example("""
+	on chat:
+		if message partially matches "\\d":
```

Comment:
do we still need the double \\ in a text block?

Reply:
Yes